pinc/graph_data.inc

```diff
@@ -730,7 +738,7 @@ function pages_daily($tally_name, $c_or_i, $timeframe)
         ],
         _("Goal") => [
             "x" => $datax,
-            "y" => $datay2,
+            "y" => is_null($datay2) ? [] : $datay2,
```

Comment:
This totally works. In PHP 7.4 you can also do:
```php
"y" => $datay2 ?? [],
```

Reply:
Your update isn't quite right.

Reply:
🤦‍♂️